database/migrations/2022_05_11_154808_config_ldap_params.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+use App\Models\Configs;
+use Illuminate\Database\Migrations\Migration;
+
+class ConfigLdapParams extends Migration
+{
+	/**
+	 * We setup the configuration for the public LDAP Server
+	 * See https://www.forumsys.com/2022/05/10/online-ldap-test-server/.
+	 *
+	 * After enabling the LDAP authentication with ldap_enabled set to 1
+	 * username: gauss and passward: password can be used.
+	 *
+	 * /**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		DB::table('configs')->insert([
+			[
+				'key' => 'ldap_enabled',
```

Comment:
It really does not matter where the configuration stuff is stored.
Since everything else like logs etc. is available from the web interface it might be convenient to have the LDAP-stuff there as well.

I would envision that in future one simply extracts the archives, sets up the web sever and everything else can be configured using the web interface.